go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/giantswarm/k8sclient/v4 v4.0.0
 	github.com/giantswarm/microerror v0.2.1
 	github.com/giantswarm/micrologger v0.3.3
+	github.com/giantswarm/to v0.3.0
```

Comment:
We are already depending on `autorest/to`. Let's use that instead of adding another dependency.